source/extensions/transport_sockets/tls/cert_validator/default_validator.h

```diff
@@ -107,18 +108,21 @@ class DefaultCertValidator : public CertValidator, Logger::Loggable<Logger::Id::
   static bool matchSubjectAltName(X509* cert,
                                   const std::vector<SanMatcherPtr>& subject_alt_name_matchers);
 
+protected:
+  // Increment failure stats and update validation state in ssl_extended_info if not nullptr.
+  void onVerifyError(Ssl::SslExtendedSocketInfo* ssl_extended_info, absl::string_view error);
+
+  bool allow_untrusted_certificate_{false};
```

Comment:
The style guide requires data members to be private, so we probably want an accessor here instead of making the member itself protected.